src/Codeception/Module/Yii2.php

```diff
@@ -223,17 +223,17 @@ public function _after(TestInterface $test)
         $_COOKIE = [];
         $_REQUEST = [];
 
+        if ($this->config['transaction'] && $this->transaction) {
+            $this->transaction->rollBack();
+            $this->debugSection('Database', 'Transaction cancelled; all changes reverted.');
+        }
+
         if ($this->config['cleanup']) {
             foreach ($this->loadedFixtures as $fixture) {
                 $fixture->unloadFixtures();
```

Comment:
ok but fixtures are cleaned up by rolled back transaction...
and this call may throw an exception...